code/controllers/subsystems/lighting.dm

```diff
@@ -54,7 +54,12 @@ SUBSYSTEM_DEF(lighting)
 
 	// Generate overlays.
 	for (var/zlevel = 1 to world.maxz)
-		overlaycount += InitializeZlev(zlevel)
+		var/datum/level_data/level = SSmapping.levels_by_z[zlevel]
+		for (var/turf/tile as anything in block(1, 1, zlevel, level.level_max_width, level.level_max_height)) // include TRANSITIONEDGE turfs
+			if (TURF_IS_DYNAMICALLY_LIT_UNSAFE(tile))
+				tile.lighting_build_overlay()
```

Comment:
Is using this helper a correctness fix? It was done manually for performance, though this is not particularly hot anyway.

Reply:
It looks like O7 just has logging in this block, I'm fine with either approach assuming calling TBO here doesn't blow out lighting init times (which I assume it doesn't).